docs/v1/use-cases.md

```diff
@@ -28,6 +28,16 @@
   from my local environment without needing to establish a formal business
   relationship with each service provider.
 
+## Searching and Browsing Services:
+    1.  As a consumer, I'm able to search my catalog for services by attributes
+        such as category
+    2.  As a consumer, I'm able to see metadata about a service prior to 
+        creation which allows me to see if this service fits my need
```

Comment:
It might be good to include something about how "price" should be included in there, but only if you're making other edits before we merge.